### ISSUE TITLE
Width doesn't work

### ISSUE THREAD

**Issue by joshangell** [engram-design/SuperTable#15]
When configuring the columns of the field there is a width option, it doesn't appear to do anything.

I took a quick look and it seems that the columns are generated using `table.getFields()` and the field model doesn't have the column width set on it so its not a simple solution of templating in the width value.

I'd suggest either using the column settings natively or perhaps extending the native `FieldModel`.

**Comment by engram-design** [comment]
Hey @joshangell the width settings are actually stored in the Super Table field settings, associated with the field id. When the input gets rendered, it _should_ fetch the ST field settings, and lookup the array for that field's id.

```
{"columns":{"531":{"width":"20"},"532":{"width":"50"}},"fieldLayout":"row","staticField":null,"selectionLabel":"Add more rows","maxRows":"5"}
```

**Comment by joshangell** [comment]
If I do `{{ dump(field) }}` in the loop at [line 13 of the tableInput template](https://github.com/engram-design/SuperTable/blob/master/supertable/templates/tableInput.html#L13-L17) I don't see that array, is that where I should be seeing those settings?

**Comment by engram-design** [comment]
Good point - the width isn't being applied to the table header, but it is on the table rows. Have a look at https://github.com/engram-design/SuperTable/blob/master/supertable/templates/fields.html#L12

**Comment by engram-design** [comment]
@joshangell Fixed in the [0.3.1](https://github.com/engram-design/SuperTable/releases/tag/0.3.1) release.
